jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/upload/FileUploadField.java

```diff
@@ -113,7 +113,6 @@ public void setValueSource(@Nullable ValueSource<byte[]> valueSource) {
 
     @Override
     public boolean isInvalid() {
-        validate();
```

Comment:
According to other fields `isInvalid()` must not trigger validation. But probably it can be breaking changes.

Reply:
mention this in the issue description